newIDE/app/src/Profile/EditProfileDialog.js

```diff
@@ -27,115 +28,122 @@ type Props = {|
   error: ?AuthError,
 |};
 
-type State = {|
-  form: EditForm,
-|};
-
 export const getUsernameErrorText = (error: ?AuthError) => {
   if (!error) return undefined;
 
   if (error.code === 'auth/username-used')
     return 'This username is already used, please pick another one.';
```

Comment:
To translate

Reply:
And it could be moved to UsernameField file just like usernameFormatErrorMessage